section3/exercises/arrays.js

```diff
@@ -106,11 +135,18 @@ var hasCharger = true;
 var chargerType = "car";
 
 // Write your conditional here
-
+//This line below is stating they can call an uber if they have greater than or equal to 15% battery
+if (percentBatteryLeft >= 15) {
+  console.log('Can call uber');
+}
+// This next line will determine you can also call an uber if you both have a charger and it is a car charger
+else if (hasCharger && chargerType) {
```

Comment:
Putting this line after the closing bracket on line 141 is the more standard syntax.